R/pkg/R/functions.R

```diff
@@ -1143,7 +1139,7 @@ setMethod("minute",
 #' @export
 #' @examples \dontrun{select(df, monotonically_increasing_id())}
 setMethod("monotonically_increasing_id",
-          signature(x = "missing"),
+          signature(),
```

Comment:
Automatic generation of S4 methods is not desirable. I hope this case can be better handled by roxygen. For now, I agree (b) is a good solution to me.